app/src/main/java/org/p2p/wallet/striga/sms/onramp/StrigaClaimSmsInputFragment.kt

```diff
@@ -21,20 +20,18 @@ import org.p2p.wallet.utils.emptyString
 import org.p2p.wallet.utils.popBackStack
 import org.p2p.wallet.utils.replaceFragment
 
-class StrigaOnRampSmsInputFragment : BaseSmsInputFragment() {
+class StrigaClaimSmsInputFragment : BaseSmsInputFragment() {
```

Comment:
I would not rename it because "Claim" is a less general term and refers specifically to the Claim button, not the process. However, if you've decided to make this change, please also rename following guys to avoid confusion:
- `SmsInputFactory.Type.StrigaOnRamp` -> `SmsInputFactory.Type.StrigaClaim`
- `StrigaOnRampSmsApiCaller` -> `StrigaClaimSmsApiCaller`
- `StrigaOnRampSmsApiCallerTest` -> `StrigaClaimSmsApiCallerTest`
- `StrigaOnRampSmsResendRequest` -> `StrigaClaimSmsResendRequest`
- `StrigaOnRampSmsVerifyRequest` -> `StrigaClaimSmsVerifyRequest`